app/views/admin/words/_word.html.erb

```diff
@@ -0,0 +1,6 @@
+    <tr>
+        <td><%= word.content %></td>
+        <td><%= word.correct_answer.content %></td>
+        <td><%= link_to "edit", edit_admin_category_word_path(@categories, word) %></td>
+        <td><%= link_to "delete", admin_category_word_path(@categories, word), method: "delete" %></td>
```

Comment:
@YutakaYamasaki Try to change format of display, something like this: 
![image](https://user-images.githubusercontent.com/20507565/45129608-551e9800-b1b7-11e8-92c9-94212dd4f24d.png)